src/components/Wiki/History/NoHistoryView.tsx

```diff
@@ -0,0 +1,43 @@
+import { LinkButton } from '@/components/Elements'
+import { Flex, Box, Image, Text } from '@chakra-ui/react'
+import React from 'react'
+import { Wiki } from '@/types/Wiki'
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
+      direction={{ base: 'column' }}
+      mt={{ base: '12', lg: '24' }}
+    >
+      <Flex
+        w={{ lg: '45%', base: '100%' }}
+        alignItems={{ base: 'center' }}
+        justifyContent={{ base: 'center' }}
```

Comment:
If you're not having different breakpoints and just `base`, then you don't need the object syntax.
```suggestion
        alignItems="center"
        justifyContent="center"
```